### ISSUE TITLE
fix: file names resulting in `-1` appearing in published URLs

### ISSUE THREAD

**Issue by bguiz** [hiero-ledger/hiero-docs#344]
## Problem

- The following files have `-1` in their suffix
- Additionally, one of them is `page-1`, which isn't a meaningful name
- This translates to URLs,
  - e.g. `getting-started/page-1.md` maps to `https://docs.hedera.com/hedera/getting-started/page-1`, which 404s (also a problem, but separate issue from this one)
  - e.g. `sdks-and-apis/sdks/tokens/enable-kyc-account-flag-1` maps to `https://docs.hedera.com/hedera/sdks-and-apis/sdks/readme-1/enable-kyc-account-flag-1`

```text
getting-started/page-1.md
sdks-and-apis/sdks/tokens/enable-kyc-account-flag-1.md
sdks-and-apis/sdks/smart-contracts/call-a-smart-contract-function-1.md
sdks-and-apis/hedera-consensus-service-api-1.md
sdks-and-apis/deprecated/sdks/tokens/enable-kyc-account-flag-1.md
```

## Solution

- Rename these files to remove the `-1` suffixes
- Update the redirects in `.gitbook.yaml` so that the previous URLs that may be linked externally still get to the intended pages

### Steps to reproduce

N/A

### Additional context

NIL

### Hedera network

other

### Version

commit 494681a8981f83fe75e87f5040299362c2d66ed3 in this repo

### Operating system

macOS

**Comment by theekrystallee** [comment]
This issue was resolved in commit 90cabf2

**Comment by bguiz** [comment]
@theekrystallee I've left a few nitpick comments on https://github.com/hashgraph/hedera-docs/commit/90cabf20e1a265074b1ce0a45b04a5815d5a7296
